pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/aktivitetsplikt/OmstillingsstoenadAktivitetspliktInformasjon4mndInnhold.kt

```diff
@@ -221,7 +221,7 @@ object OmstillingsstoenadAktivitetspliktInformasjon4mndInnhold :
                 }
             }
 
-            showIf(aktivitetsgrad.equalTo(Aktivitetsgrad.OVER_50_PROSENT) or not(utbetaling)) {
+            showIf(aktivitetsgrad.equalTo(Aktivitetsgrad.OVER_50_PROSENT)) {
```

Comment:
Blir det evt slik? Hvis den skal gjelde for alt over 50 %? (er ikke sikker på enumen)

```suggestion
            showIf(aktivitetsgrad.equalTo(Aktivitetsgrad.OVER_50_PROSENT) or aktivitetsgrad.equalTo(Aktivitetsgrad.HUNDRE_PROSENT)) {
```